From: Suren Baghdasaryan <surenb@google.com>
Subject: fixup! increase PERCPU_MODULE_RESERVE to accommodate allocation tags
Date: Sat, 6 Apr 2024 14:40:43 -0700

The increase of per-cpu reserved area for modules was not enough to
accommodate all allocation tags in certain use cases.  Increase some more
to fix the issue.

Link: https://lkml.kernel.org/r/20240406214044.1114406-1-surenb@google.com
Fixes: a11cb5c8e248 ("mm: percpu: increase PERCPU_MODULE_RESERVE to accommodate allocation tags")
Signed-off-by: Suren Baghdasaryan <surenb@google.com>
Reported-by: Klara Modin <klarasmodin@gmail.com>
Tested-by: Klara Modin <klarasmodin@gmail.com>
Cc: Kent Overstreet <kent.overstreet@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/percpu.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/percpu.h~mm-percpu-increase-percpu_module_reserve-to-accommodate-allocation-tags-fix
+++ a/include/linux/percpu.h
@@ -15,7 +15,7 @@
 /* enough to cover all DEFINE_PER_CPUs in modules */
 #ifdef CONFIG_MODULES
 #ifdef CONFIG_MEM_ALLOC_PROFILING
-#define PERCPU_MODULE_RESERVE		(8 << 12)
+#define PERCPU_MODULE_RESERVE		(8 << 13)
 #else
 #define PERCPU_MODULE_RESERVE		(8 << 10)
 #endif
_
